Project-Hotpot/Project-Hotpot/PFSong.swift

```diff
@@ -12,14 +12,29 @@ import Parse
 class PFSong: PFObject, PFSubclassing {
     @NSManaged var name : String
     @NSManaged var URI: String
-    @NSManaged var duration: Int
+    @NSManaged var duration: UInt
     @NSManaged var artist: String
     @NSManaged var album: String
     
     static func parseClassName() -> String {
         return "Song"
     }
-
+    
+    class func saveSong(song:SPTAppRemoteTrack) {
```

Comment:
Hmm looks like this method does not convey that song will be saved in background. Check with @toby1236  for best practice, for something similar on Android the consumer will call save on a background thread to make it very apparent in the UI layer code. We also have annotations that force devs at compile time to call methods on a background thread.

Reply:
I agree though with propagating some indicator that this will finish "sometime later" and not immediately, though, to convey to other engineers that the saved song may not be immediately available right after this call.